core/variant/variant_parser.cpp

```diff
@@ -1123,13 +1124,46 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 				get_token(p_stream, token, line, r_err_str);
 				if (token.type == TK_STRING) {
 					String path = token.value;
-					Ref<Resource> res = ResourceLoader::load(path);
+					String uid_string;
+
+					get_token(p_stream, token, line, r_err_str);
+
+					if (path.begins_with("uid://")) {
+						uid_string = path;
+						path = "";
+					}
+					int which_arg = 1;
+					while (token.type == TK_COMMA) {
```

Comment:
This will keep scanning for strings, and anything after first argument will be discarded. What's the purpose of that?

Reply:
I was thinking of this in terms of an argument list, but that was too general purpose.